playbooks/byo/openshift-cluster/config.yml

```diff
@@ -15,6 +15,11 @@
       checks:
       - disk_availability
       - memory_availability
+      - package_availability
+      - package_update
+      - package_version
+      - docker_image_availability
+      - docker_storage
```

Comment:
aos-ci-test does not have proper docker storage set up so I had it disabled for now (and it passes). Regular CI showed I needed to take `oreg_url` into account but once I did that it passed fine.